app/views/lessons/_advert.html.erb

```diff
@@ -1,32 +1,38 @@
-<div class="card-main advert">
-  <div class="advert__header">
-    <h2 class="advert__pitch">Are you interested in accelerating your web development learning experience?</h2>
-    <%= link_to "https://www.thinkful.com/?utm_source=odin&utm_medium=cpc&utm_campaign=odin-homepage&utm_content=web-dev-3-icons", class: "button button--primary ad"  do%>
-      Get started
-    <% end %>
-  </div>
-  <div class="advert__body">
-    <a href="https://www.thinkful.com/?utm_source=odin&utm_medium=cpc&utm_campaign=odin-homepage&utm_content=tf-logo" alt="Thinkful" class="advert__brand-anchor ad">
-        <div class="advert__brand">
-          <%= image_tag 'advert/thinkful-logo.svg', alt: 'Thinkful Logo', class: 'advert__logo' %>
-          <h2 class="advert__brand-name">Thinkful</h2>
-        </div>
+<%= render CardComponent.new(classes: 'odin-dark-bg-accent') do |card| %>
+  <% card.header do %>
+    <div class="flex justify-between gap-x-16 items-center border-b-2 border-gray-200 pb-6 flex-col space-y-6 md:flex-row md:space-y-0">
+      <h2 class="text-2xl text-gray-500 text-center w-2/3 md:text-left">
+        Are you interested in accelerating your web development learning experience?
+      </h2>
+      <%= link_to "https://www.thinkful.com/?utm_source=odin&utm_medium=cpc&utm_campaign=odin-homepage&utm_content=web-dev-3-icons", class: "button button--primary" do %>
```

Comment:
Do these button styles need to be changed now, or will that be sorted in a separate story?

Reply:
Sorry mate, I'm probably missing something right in front of me haha. What needs changed about them?

Reply:
@kevinmulhern No worries mate, I should have been more specific :sweat_smile: 
The BEM classes right at the end, `button button--primary`

Reply:
Ahh I gotcha now, the buttons are already in Tailwind https://github.com/TheOdinProject/theodinproject/blob/main/app/javascript/layouts/buttons.css